tidb-cloud/ticloud-config-edit.md

```diff
@@ -0,0 +1,43 @@
+---
+title: ticloud config edit
+Summary: The reference of `ticloud config edit`.
+---
+
+# ticloud config edit
+
+If you are using macOS or Linux, you can open the profile configuration file with your default text editor:
+
+```shell
+ticloud config edit [flags]
+```
+
+If you are using Windows, after you execute the preceding command, the path of the profile configuration file will be printed instead.
+
+> **Note:**
+>
+> To avoid format errors and executions failures, it is NOT recommended to manually edit the configuration file. Instead, you can use [`ticloud config create`](/tidb-cloud/ticloud-config-create.md), [`ticloud config delete`](/tidb-cloud/ticloud-config-delete.md), or [`ticloud config set`](/tidb-cloud/ticloud-config-set.md) to modify the confiturations.
```

Comment:
```suggestion
> To avoid format errors and execution failures, it is NOT recommended to manually edit the configuration file. Instead, you can use [`ticloud config create`](/tidb-cloud/ticloud-config-create.md), [`ticloud config delete`](/tidb-cloud/ticloud-config-delete.md), or [`ticloud config set`](/tidb-cloud/ticloud-config-set.md) to modify the confiturations.
```